geode-docs/managing/disk_storage/backup_restore_disk_store.html.md.erb

```diff
@@ -38,6 +38,15 @@ Do not try to create backup files from a running system by using your operating
 **Preparing to Make a Backup**
 
 -   Consider compacting your disk store before making a backup. If auto-compaction is turned off, you may want to do a manual compaction to save on the quantity of data copied over the network by the backup. For more information on configuring a manual compaction, see [Manual Compaction](compacting_disk_stores.html#compacting_disk_stores__li_63CF8C35153D4173AADF7DC35FEC61F9).
+-   Take the backup when region operations are quiescent,
+to avoid the possibility of an inconsistency between region data and
+an asyncronous event queue (AEQ),
+if that backup were to be used to restore a system.
+The AEQ is persistent.
+If a backup is unlucky enough to be taken between
```

Comment:
I think Diane captured some of this already.

I'm not sure I understand the formatting in this passage. I'm looking at it on github, though, so maybe thats why. I'm not sure I understand this sentence: "The AEQ is persistent." Is it meant to emphasize both the region and queue are persistent and writes to them are not atomic?

asyncronous -> asynchronous

Its not just AEQs that could be inconsistent. Also WAN GatewaySenders. They both use the same queueing code underneath the covers. Its the persistent queue that could be out of sync with the persistent data region, so both AEQs and WAN GatewaySenders can have the same issue.